src/snake.rs

```diff
@@ -47,14 +38,14 @@ impl Snake {
     }
 
     pub fn on_game_starting(&self, _: &GameStarting) {
-
+        debug!(target: LOG_TARGET, "All snakes are ready to rock. Game is starting.");
     }
 
     pub fn on_player_registered(&self, _: &PlayerRegistered) {
-
+        debug!(target: LOG_TARGET, "Player has been registered.");
     }
 
     pub fn on_invalid_playername(&self, _: &InvalidPlayerName) {
-
+        debug!(target: LOG_TARGET, "I'm sorry, I cannot let you do that Dave.");
```

Comment:
While these are clever I think it is generally better to stick to boring but informative defaults for these, i.e.:
`{:?} is not a valid player name!`